ACD-315 (#5)
Actindo-AG/Documentation: PIM/Operation/03_MoveProducts.md

[!!User Interface Products](../UserInterface/02_Products.md)   
[!!Manage the products](./01_ManageProducts.md)

# Move a product

> Move a product to either archive it, move it to the recycle bin, delete it finally or restore it.

Reviewer: ```suggestion
Move a product to either archive it, move it to the recycle bin, delete it permanently or restore it.
```

## Archive a product

> You can archive products that you are not currently using, but that you still want to access and may want to use again at a later time. Archived products cannot be finally deleted but must first be moved to the recycle bin.

Reviewer: s.o. -> Ich habe überall finally durch permanently ersetzt (vielleicht persönliches Geschmack).

#### Prerequisites

At least one product has been created, see [Create a product](./01_ManageProducts.md#create-a-product).

#### Procedure
*PIM > Products > Tab LIST*    
*PIM > Products > Tab RECYCLE BIN*

![List and recycle bin](../../Assets/Screenshots/PIM/Products/ListRecycleBin.png "[List and recycle bin]")

> [Info] You can archive products from the product list or from the recycle bin. The procedure to archive them is identical. To archive a product from the recycle bin, start in the *RECYCLE BIN* tab instead of in the *LIST* tab and follow the steps described below.

> 1. Select the checkboxes of the products you want to archive.   

Reviewer: ```suggestion
1. Select the checkbox(es) of the product(s) you want to archive.   
```

#### Prerequisites

At least one product has been created, see [Create a product](./01_ManageProducts.md#create-a-product).

#### Procedure
*PIM > Products > Tab LIST*   
*PIM > Products > Tab ARCHIVE*

![List and archive](../../Assets/Screenshots/PIM/Products/ListArchive.png "[List and archive]")

> [Info] You can delete products from the product list or from the archive. The procedure to delete them is identical. To delete a product from the archive, start in the *ARCHIVE* tab instead of in the *LIST* tab and follow the steps described below.

1. Select the checkboxes of the products you want to delete.   
  The editing toolbar is displayed above the product list.

2. Click the [MOVE TO RECYCLE BIN] button in the toolbar.   
  The button changes to [DELETING...]. The *Product deleted* pop-up window is displayed, indicating the number of deleted products.

  ![Product deleted](../../Assets/Screenshots/PIM/Products/List/ProductDeleted.png "[Product deleted]")

  The selected products have been moved to the recycle bin.

>   > [Info] You find the deleted products in the *RECYCLE BIN* tab where you can either restore or finally delete them.

Reviewer: s.o.

## Finally delete a product

> You can permanently delete products Products that have been moved to the recycle bin unless they have any existing dependencies. Be aware that once deleted products cannot be restored as the deletion process cannot be undone.

Reviewer: ```suggestion
You can permanently delete products that have been moved to the recycle bin unless they have any existing dependencies. Be aware that, once permanently deleted, products cannot be restored as the deletion process cannot be undone.
```

#### Prerequisites

At least one product has been moved to the recycle bin, see [Delete a product](#delete-a-product).

#### Procedure
*PIM > Products > Tab RECYCLE BIN*

![Recycle bin](../../Assets/Screenshots/PIM/Products/RecycleBin/RecycleBin.png "[Recycle bin]")

1. Select the checkboxes of the products you want to delete.   
  The editing toolbar is displayed above the product list.

2. Click the [FINALLY DELETE PRODUCTS] button in the toolbar.   
  The button changes to [DELETING...]. The *Products finally deleted* pop-up window is displayed, indicating the number of finally deleted products.

  ![Product finally deleted](../../Assets/Screenshots/PIM/Products/RecycleBin/ProductFinallyDeleted.png "[Product finally deleted]")

  The selected products have been permanently deleted.

>   > [Info] If the deletion is not possible, the *Products deleted* pop-up window is indicating an error for the number of products unable to delete. An error message with further information is displayed in the notifications.

Reviewer: ```suggestion
  > [Info] If the deletion is not possible, the *Products deleted* pop-up window indicates an error for the number of products that cannot be deleted. An error message with further information is displayed in the notifications.
```
Wo ist "notifications"? Evtl. Link?
